### PR TITLE
#140: Coverage improvements

### LINKED ISSUE #140: Increase Code Coverage
At the Moment we have around 97% Code Coverage, obviously that isn't bad, but we should be able to reach 99% or 100% Coverage. There are just some new methods which have no tests yet.

The Uncovered Methods (at the time of writing):

Screen Shot 2020-08-01 at 12 35 29 PM

_Both the Coverage and Screenshot are from the soon-to-be-merged `fix-lints-janni-ben` branch_

### PR DESCRIPTION
**Pull request by sinnaj-r**

### CODE REVIEW

**Comment by sinnaj-r** [comment]
We now have (in the Squeak TestRunner) 100% Coverage 🎉

File: packages/AlgernonTests.package/ALGTest.class/instance/prevAlgernonSingletone..st [hunk 1]
```diff
@@ -0,0 +1,3 @@
+accessing
+prevAlgernonSingletone: anObject
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTest.class/instance/prevAlgernonSingletone..st]: I think there is a typo here. It should be `Singleton`

Comment (BenBals) [reply packages/AlgernonTests.package/ALGTest.class/instance/prevAlgernonSingletone..st]: Also we have another instance variable starting with `previous` maybe we should be consistent.

File: packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonChangeShortcut..st [hunk 2]
```diff
@@ -1,6 +1,6 @@
 testing
 testAlgernonChangeShortcut: shortcut 
-	ALGCore activationString: shortcut shallowCopy.
+	ALGCore setKeyboardBinding: shortcut shallowCopy.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonChangeShortcut..st]: Is here a reason for `shallowCopy` here. `Object>>copy` is defined as `self shallowCopy postCopy`.

File: packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonSingletoneClose.st [hunk 3]
```diff
@@ -0,0 +1,5 @@
+testing
+testAlgernonSingletoneClose
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonSingletoneClose.st:2]: Same typo.

File: packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonSingletoneOpen.st [hunk 4]
```diff
@@ -0,0 +1,5 @@
+testing
+testAlgernonSingletoneOpen
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonSingletoneOpen.st:2]: Same typo

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testCategoryTypeOpenMethodInBrowser.st [hunk 5]
```diff
@@ -0,0 +1,8 @@
+testing category type
+testCategoryTypeOpenMethodInBrowser
+	| browser category|
+	browser :=SystemBrowser default open.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testCategoryTypeOpenMethodInBrowser.st]: Spacing.

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testCategoryTypeOpenMethodInBrowser.st [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+	category:= 'Algernon'.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testCategoryTypeOpenMethodInBrowser.st]: Spacing.

Maybe just run ohm pretty print?

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testHelpAction.st [hunk 7]
```diff
@@ -0,0 +1,5 @@
+testing help type
+testHelpAction
+	self runQuery: 'help browse'.
+	self algInstance doActionOnSelectedEvent: nil.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testHelpAction.st]: Use `self activateCurrentItemAction`.

File: packages/AlgernonTests.package/ALGTestTypes.class/instance/testSystemCommandDo.st [hunk 8]
```diff
@@ -0,0 +1,6 @@
+testing system command
+testSystemCommandDo
+	| called |
+	called:=false.
```
Comment (BenBals) [inline packages/AlgernonTests.package/ALGTestTypes.class/instance/testSystemCommandDo.st]: Spacing.

**Review by BenBals — CHANGES_REQUESTED** [review]
Looks good, just a few nitpicks.

File: README.md [hunk 9]
```diff
@@ -3,7 +3,7 @@
 Master: [![Coverage Master](https://coveralls.io/repos/github/hpi-swa-teaching/Algernon-Launcher/badge.svg?branch=master)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master) Development: [![Coverage Development](https://coveralls.io/repos/github/hpi-swa-teaching/Algernon-Launcher/badge.svg?branch=development)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master) 
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/44369294/89326002-e93a5800-d689-11ea-89f0-d685e7a4f1e9.png" width="465" alt="Algernon"/>
```
Comment (BenBals) [inline README.md]: Please revert this merge.

**Review by BenBals — CHANGES_REQUESTED** [review]
Lgtm

**Review by BenBals — APPROVED** [review]
Yay! :rocket:

**Pull request merged**